packages/@o3r/components/builders/component-extractor/helpers/component/component-class.extractor.ts

```diff
@@ -57,32 +63,28 @@ export class ComponentClassExtractor {
 
   /**
    * Get the component type from the given decorator node.
-   *
    * @param decoratorNode The decorator node to get the component type from
    */
   private getComponentType(decoratorNode: ts.Decorator): ComponentStructure | undefined {
-    if (ts.isCallExpression(decoratorNode.expression)) {
-      if (decoratorNode.expression.expression.getText(this.source) === 'O3rComponent') {
-        const arg1 = decoratorNode.expression.arguments[0];
-        if (ts.isObjectLiteralExpression(arg1)) {
-          return this.getComponentStructure(
-            arg1.properties
-              .find((prop): prop is ts.PropertyAssignment => prop.name?.getText(this.source) === 'componentType')
-              ?.initializer.getText(this.source)
-          );
-        }
+    if (ts.isCallExpression(decoratorNode.expression) && decoratorNode.expression.expression.getText(this.source) === 'O3rComponent') {
+      const arg1 = decoratorNode.expression.arguments[0];
+      if (ts.isObjectLiteralExpression(arg1)) {
+        return this.getComponentStructure(
+          arg1.properties
+            .find((prop): prop is ts.PropertyAssignment => prop.name?.getText(this.source) === 'componentType')
+            ?.initializer.getText(this.source)
+        );
       }
     }
   }
 
   /**
    * Get the component selector from the given decorator node.
-   *
    * @param decoratorNode The decorator node to get the component selector from
    */
   private getComponentSelector(decoratorNode: ts.Decorator) {
     if (this.isComponentDecorator(decoratorNode)) {
-      const matches = /selector:\s*['"](.*)['"]/.exec(decoratorNode.getText(this.source));
+      const matches = /selector:\s*["'](.*)["']/.exec(decoratorNode.getText(this.source));
       if (matches) {
```

Comment:
Why it has not be fixed by auto fix run earlier?

Reply:
As I said in our chat I don't know why the auto fix didn't run on this package earlier :/